pkg/cpi/mockingUtils.go

```diff
@@ -55,17 +55,17 @@ func GetCPIFunctionMockResponse(functionName, testType string) (*http.Response,
 	}
 }
 
-//GetEmptyHTTPResponseBody -Empty http respose body
-func GetEmptyHTTPResponseBody() (*http.Response, error) {
+//GetEmptyHTTPResponseBodyAndNilError -Empty http respose body
+func GetEmptyHTTPResponseBodyAndNilError() (*http.Response, error) {
 	res := http.Response{
 		StatusCode: 202,
 		Body:       ioutil.NopCloser(bytes.NewReader([]byte(``))),
 	}
 	return &res, nil
 }
 
-//GetNegativeCaseHTTPResponseBody -Negative case http respose body
-func GetNegativeCaseHTTPResponseBody() (*http.Response, error) {
+//GetNegativeCaseHTTPResponseBodyAndErrorNil -Negative case http respose body
+func GetNegativeCaseHTTPResponseBodyAndErrorNil() (*http.Response, error) {
```

Comment:
These were just name suggestions from my side, please choose between `ErrorNil` and `ErrorNil` suffix for the two functions.

Reply:
Sorry typo. I meant `ErrorNil` or `NilError`